rustworkx-core/src/generators/hexagonal_lattice_graph.rs

```diff
@@ -17,19 +17,200 @@ use petgraph::visit::{Data, NodeIndexable};
 
 use super::InvalidInputError;
 
+mod utils {
```

Comment:
I don't really think we need this module, it could be include with the rest of the code. But that is a minor detail for now